teachopencadd/talktorials/T013_query_pubchem/talktorial.ipynb

```diff
@@ -6,10 +6,13 @@
    "source": [
```

Comment:
<p>Run</p><pre class="ql-syntax" spellcheck="false">black -l 99 your-talktorial.ipynb
</pre><p>That should take care e.g. of your mix of "" and '' string quotes in e.g. line 9</p><p> </p><p>Use f-strings?</p><p><code>print("Molecular weight for Aspirin is:\n{mol_weight}")</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/volkamerlab/teachopencadd/pull/81/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>

Reply:
<p>done</p>

Reply:
<pre class="ql-syntax" spellcheck="false">black -l 99 your-talktorial.ipynb
</pre><p>just ruins this notebook. </p>

Reply:
<p>Oh no, in what way?</p><p> </p><p>Maybe you copied over my typo?</p><p><code>print("Molecular weight for Aspirin is:\n{mol_weight}")</code></p><p>should be</p><p><code>print(<strong>f</strong>"Molecular weight for Aspirin is:\n{mol_weight}")</code></p><p>I forgot to add the <strong>f</strong> in front of the string, sorry!</p>

Reply:
<p>Command is <code>black-nb</code>  !</p>

Reply:
<p>I have a small problem when I try to use "black-nb". It always report: "AttributeError: module 'black' has no attribute 'PY36_VERSIONS'" even after I change the python env to python3.6. And this problem happened both in MacOS and Ubuntu.&nbsp;</p>

Reply:
Hm, intriguing. I can run it for you. Just address the other feedback for now, no worries!